src/lib/skip-link/skip-link.ts

```diff
@@ -0,0 +1,103 @@
+import { attachShadowTemplate, coerceBoolean, coreProperty, customElement } from '@tylertech/forge-core';
+import { BaseComponent, IBaseComponent } from '../core';
+import { FocusIndicatorComponent } from '../focus-indicator';
+import { StateLayerComponent } from '../state-layer';
+import { SkipLinkAdapter } from './skip-link-adapter';
+import { SKIP_LINK_CONSTANTS, SkipLinkTheme } from './skip-link-constants';
+import { SkipLinkCore } from './skip-link-core';
+
+import template from './skip-link.html';
+import style from './skip-link.scss';
+
+export interface ISkipLinkComponent extends IBaseComponent {
+  target: string;
+  theme: SkipLinkTheme;
+  muted: boolean;
+  persistent: boolean;
+}
+
+declare global {
+  interface HTMLElementTagNameMap {
+    'forge-skip-link': ISkipLinkComponent;
+  }
+}
+
+/**
+ * @tag forge-skip-link
+ *
+ * @summary The Forge Skip Link component is used to provide a way for users to skip repetitive content and navigate directly to a section of the page.
+ *
+ * @prop {string} [target=''] - The IDREF of the element to which the skip link should navigate.
```

Comment:
Can you update this to `@property` for consistency?

Also, let's add `@dependency` tags for the state layer and focus indicator as well.